_benchmark/user-guide/contributing-workloads.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Contributing custom workloads
+nav_order: 11
+parent: User guide
+---
+
+# Contributing custom workloads
+
+After [Creating a custom workload]({{site.url}}{{site.baseurl}}/benchmark/creating-custom-workloads/) you can contribute your workload for other OpenSearch Benchmark users to use. If you plan on configuring a workload, use the following guidelines.
```

Comment:
- "Creating a custom workload" --> "creating a custom workload" (uppercased C needs to be lowercased)
- "configuring a workload" --> "contributing a workload"